docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -0,0 +1,106 @@
+# Authenticating with client certificates
+
+You can perform authentication with client certificates.
+
+
+:::note
+There is a limitation with respect to performing authentication using Z Secure Services (ZSS) with ACF2 systems. If you are using ACF2, and are using Zowe v2.14 or later, use the recommended the internal API ML mapper.
+:::
+
+When providing credentials during a login request with a client certificate on the same login request, the login credentials take precedence and the client certificate is ignored.
```

Comment:
shouldn't this be a part of a `:::note` as well?

Reply:
good catch. done